scripts/box

```diff
@@ -177,14 +177,30 @@ We URGE you to migrate to a supported release if/while you still have the chance
 
 function _upgrade() {
     arg=("$@")
-    packages=("${arg[@]:1}")
+    if [[ "${arg[*]}" == *--all* ]]; then
+        package_string="$(find /install -type f -name .*.lock | awk -F. '{print $2}')"
+        blacklist=("rtorrent" "qbittorrent" "deluge" "libtorrent" "panel" "transmission" "curl" "nginx")
+        for blocker in "${blacklist[@]}"; do
+            package_string="$(echo "$package_string" | sed "/$blocker/d")"
+        done
+        echo_info "Upgrading all packages, EXCEPT torrent clients and packages with alternative upgrade methods (apt/built-in)\nAttempting packages: '${package_string/$'\n'/, }'"
+        readarray -t packages < <(echo "$package_string")
+    else
+        packages=("${arg[@]:1}")
+    fi
+
     repo=($(find /usr/local/bin/swizzin/upgrade -type f -printf "%f\n" | cut -d "." -f 1))
     for i in "${packages[@]}"; do
         app=${i}
```

Comment:
This seems unused

Reply:
I don't see it used directly but it's not something I changed in this PR. This could be a var used in one of the sub-scripts possibly? I don't see how that could be the case but I prefer not to touch that rn.